include jupyter notebooks in docs (#229)
aiidalab/aiidalab-widgets-base: docs/source/widget-list/index.rst

List of widgets
***************

> Here we collect a set of Jupyter notebooks that illustrate how to use all available widgets.

Reviewer: ```suggestion
This is a collection of Jupyter notebooks that illustrate how to use the widgets available from this library.
```
